pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -153,7 +153,7 @@ type InitializerSpec struct {
 // +k8s:openapi-gen=true
 // MonitorContainer is the common attributes of the container of monitoring
 type MonitorContainer struct {
-	Resources corev1.ResourceRequirements `json:",inline"`
+	corev1.ResourceRequirements `json:",inline"`
```

Comment:
Keep the same as TidbCluster: https://github.com/pingcap/tidb-operator/blob/v1.1.2/pkg/apis/pingcap/v1alpha1/types.go#L285